apps/amiapp_flutter/ios/Podfile

```diff
@@ -57,6 +57,8 @@ target 'NotificationServiceExtension' do
   pod 'customer_io_richpush/fcm', :path => '.symlinks/plugins/customer_io/ios' # install podspec bundled with the plugin
   # install_non_production_ios_sdk_local_path(local_path: '~/code/customerio-ios/', is_app_extension: true, push_service: "fcm")
   # install_non_production_ios_sdk_git_branch(branch_name: 'levi/v2-multiple-push-handlers', is_app_extension: true, push_service: "fcm")
+  # To avoid sample app breaking on CI with newer versions of Firebase (11.12.0+) until it is fixed in native iOS SDK
+  pod 'FirebaseCoreInternal', '~> 11.11.0'
```

Comment:
wonder which dependency is bringing it?

Reply:
I think the other dependencies we've added for Firebase in sample app are likely causing newer versions to be fetched. Though I'm not entirely sure since `Podfile.lock` didn't clearly point to anything other than `FirebaseCoreInternal` to help identify parent dependency pulling it in.